fendermint/app/src/cmd/key.rs

```diff
@@ -66,6 +70,27 @@ cmd! {
   }
 }
 
+cmd! {
+  AddPeer(self) {
+    let node_key = NodeKey::load_json_file(&self.node_key_file).unwrap();
+    let peer_id : String = node_key.node_id().to_owned().to_string()+"@"+self.network_addr.as_str();
+    let peers = std::fs::read_to_string(&self.local_peers_file).context("failed to read peer file");
+    match peers {
+        Ok(mut v) => {
+            if v.is_empty()  {
+                std::fs::write(&self.local_peers_file, peer_id)?;
```

Comment:
It would be easier to follow what's going on if the file was only written to in one place:
1. Read the file into a string, or get the default one if it doesn't exist.
2. Modify the string by appending the new peer, with or without a comma (do you care about duplicates?)
3. Write the string to the file.

Reply:
Fixed